### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by YanaKenda**
Description of use-case, activities, db_diagram, class_diagram.

### CODE REVIEW

**Comment by tkarpesh** [comment]
:+1: :blush: 
Please, remove the Uml modeling and documentation/Doc

**Comment by oleguchok** [comment]
Remove Uml modeling and documentation/Doc.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# spp-company
```
Comment (tkarpesh) [inline README.md]: test

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## Sphere: 
+
+The sphere is driving school. This app is automated service workflow. Service is designed to optimize and improve the efficiency of driving school process, helping to reduce costs and encrease profits. 
+
+## Actors:
+
+* Guest
+* User
```
Comment (oleguchok) [inline README.md]: Remove this line

File: README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+* Student
+* Teacher
+* Manager
+* Administator
+
+## Entities:
```
Comment (oleguchok) [inline README.md]: I think that this Entities list is not necessary

File: README.md (hunk 2 above)
Comment (tkarpesh) [inline README.md]: :+1:

**Comment by YanaKenda** [comment]
Oleg, can you explain your thoughts as Tanya said.

File: README.md (hunk 2 above)
Comment (tkarpesh) [inline README.md]: Remove this line means to remove the line with the word ``User``

Comment (oleguchok) [inline README.md]: @tkarpesh , yes!

**Pull request merged**